src/corona/emissivity.jl

```diff
@@ -97,24 +106,38 @@ This function assumes axis symmetry, and therefore always interpolates the emiss
 as a function of the radial coordinate on the disc. If non-symmetric profiles are 
 desired, consider using [`tracecorona`](@ref) with a profile constructor, e.g. [`VoronoiDiscProfile`](@ref).
 """
-emissivity_profile(
+function emissivity_profile(
     m::AbstractMetric,
     d::AbstractAccretionGeometry,
     model::AbstractCoronaModel;
+    spectrum = PowerLawSpectrum(2.0),
+    kwargs...,
+)
+    emissivity_profile(m, d, model, spectrum; kwargs...)
+end
+function emissivity_profile(
+    m::AbstractMetric,
+    d::AbstractAccretionGeometry,
+    model::AbstractCoronaModel,
+    spectrum::AbstractCoronalSpectrum;
     sampler = nothing,
     kwargs...,
-) = emissivity_profile(sampler, m, d, model; kwargs...)
+)
+    emissivity_profile(sampler, m, d, model, spectrum; kwargs...)
+end
```

Comment:
Bootstrapping in the ability to leave out the spectrum from the arguments. This can and probably will be cleaned up cus, as you can see, it's a little headachey to read.